scripts/check-translations.ts

```
@@ -0,0 +1,43 @@
+
+const enKeys = new Set(getLeafKeys(en as unknown as JsonObject));
+const esKeys = new Set(getLeafKeys(es as unknown as JsonObject));
+
+const missingInEs = [...enKeys].filter((k) => !esKeys.has(k));
+const extraInEs = [...esKeys].filter((k) => !enKeys.has(k));
+
+let hasError = false;
+
+if (missingInEs.length > 0) {
+  console.error('❌ Keys in en.json missing from es.json:');
+  missingInEs.forEach((k) => console.error(`  - ${k}`));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)